src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/GetSecurityGroupsIntTest.java

```diff
@@ -45,43 +39,6 @@ class GetSecurityGroupsIntTest extends IntegrationBase {
     @Autowired
     private MockMvc mockMvc;
 
-    @Test
```

Comment:
Why was this one removed?

Reply:
Removed because there is already an int test that covers it - `SecurityGroupControllerIntTest.getSecurityGroupsShouldSucceedAndReturnAllGroups()`. And there's also a functional test that covers it - `SecurityGroupFunctionalTest.shouldGetSecurityGroups()`